src/scales/categoryScale.ts

```diff
@@ -142,7 +142,14 @@ module Plottable.Scales {
     }
 
     protected _getDomain() {
-      return this._d3Scale.domain();
+      let newlineRemover = function(domain) {
+        if (typeof domain !== "string") {
+          return domain;
+        }
+        return domain.replace(/\n/g, " ");
+      };
+
+      return this._d3Scale.domain().map(newlineRemover);
```

Comment:
I'm not sure it's a good idea to modify the domain. This change would produce some unexpected behavior:
```Javascript
var originalDomain = ["two\nlines"];
categoryScale.domain(originalDomain);
assert.deepEqual(categoryScale.domain(), originalDomain); // fails
```